test/unit/shadow/polyfill.js

```diff
@@ -81,6 +81,13 @@ describe('shadow/polyfill', function () {
     it('innerHTML', function () {
       root.innerHTML = '<slot name="test"></slot>';
       expect(root.childNodes[0].name).to.equal('test');
+
+      if (canPatchNativeAccessors) {
```

Comment:
In Safari `__innerHTML` isn't available because they don't return native descriptors from `Object.getOwnPropertyDescriptor()`.